resource/authentication_manager.go

```diff
@@ -17,22 +18,25 @@ import (
 )
 
 type AuthenticationManager struct {
-	AuthPath string
+	CSSAuthPath      string
```

Comment:
I was envisioning that the secrets would be managed by a separate object, not the auth manager itself. I think we'd like to keep those components separated.

Reply:
Separated the secrets manager out from auth manager